### ISSUE TITLE
Shared logic between XE and XR scripts where possible

### ISSUE THREAD

**Issue by glennmatthews** [ios-xr/iosxrv-x64-vbox#25]
Currently the iosxr_iso2vbox.py and iosxe_iso2vbox.py scripts have a lot of duplicate code. So that each script can benefit from bugfixes and enhancements to the other, we should create a shared library for any code that can reasonably be shared between the two scripts.

**Comment by xiaopeng163** [comment]
agree, also for #23, it's better if this tool can be provided as a python package.
